build/KoreBuild.sh

```diff
@@ -119,24 +119,34 @@ fi
 
 makeFileProj="$koreBuildFolder/KoreBuild.proj"
 msbuildArtifactsDir="$repoFolder/artifacts/msbuild"
+msbuildPreflightResponseFile="$msbuildArtifactsDir/msbuild.preflight.rsp"
 msbuildResponseFile="$msbuildArtifactsDir/msbuild.rsp"
 msbuildLogFile="$msbuildArtifactsDir/msbuild.log"
 
 if [ ! -f $msbuildArtifactsDir ]; then
     mkdir -p $msbuildArtifactsDir
 fi
 
+cat > $msbuildPreflightResponseFile <<ENDMSBUILDPREFLIGHT
+/nologo
+/p:NetFxVersion=$netfxversion
+/p:PreflightRestore=true
+/p:RepositoryRoot="$repoFolder/"
+/t:Restore
+"$makeFileProj"
+ENDMSBUILDPREFLIGHT
+
+__exec dotnet msbuild @"$msbuildPreflightResponseFile"
+
 cat > $msbuildResponseFile <<ENDMSBUILDARGS
 /nologo
 /m
-"$makeFileProj"
 /p:RepositoryRoot="$repoFolder/"
-/p:NetFxVersion=$netfxversion
 /fl
 /flp:LogFile="$msbuildLogFile";Verbosity=detailed;Encoding=UTF-8
 /clp:Summary
+"$makeFileProj"
 ENDMSBUILDARGS
 echo -e "$msbuild_args" >> $msbuildResponseFile
 
-__exec dotnet msbuild /nologo /t:Restore /p:PreflightRestore=true $makeFileProj
```

Comment:
A simpler fix would have been to simply add `/p:NetFxVersion=$netfxversion` here